sentry_streams/sentry_streams/examples/events.py

```diff
@@ -78,16 +81,18 @@ def __init__(self) -> None:
         self.alert_type: str
         self.alert_id: int
 
-    def add(self, value: TimeSeriesDataPoint) -> Self:
-        if value.alert_type == "count":
-            self.count += 1
-            self.alert_type = value.alert_type
-            self.alert_id = value.alert_id
+    def add(self, message: Message[list[TimeSeriesDataPoint]]) -> Self:
```

Comment:
Not sure I get why we end up having a `list[TimeSeriesDataPoint]`.
IF I remember correctly, we map, then flatmap to produce messages each of which contains a `TimeSeriesDataPoint`. Is the list needed because flatMap does not work ?